### PR TITLE
travisCI: Explicitly print test-bot logs, pipe dependency log to file

### PR DESCRIPTION
**Pull request by saketkc**
In continuation to #1679(https://github.com/Homebrew/homebrew-science/pull/1679#issuecomment-69481741)
this commit explicitly prints the output of test-bot
while building dependency is still silent.

### CODE REVIEW

File: travisCI/travis.sh [hunk 1]
```diff
@@ -40,7 +41,10 @@ for file in $changed_files
 do
     if [[ ${file: -3} == ".rb" ]]
     then
-        brew test-bot $file -v >> $BUILD_OUTPUT 2>&1
+        # Dump output of building dependencies to log file
+        brew install $(brew deps $file) >> $BUILD_OUTPUT 2>&1
```
Comment (dpo) [inline travisCI/travis.sh]: Let's use `reinstall` or else `brew` throws an error if a dep is already there.

**Comment by saketkc** [comment]
I have updated it.

**Comment by dpo** [comment]
Thanks!

**Comment by saketkc** [comment]
@dpo I am having trouble with the reinstall command.
See: https://travis-ci.org/saketkc/linuxbrew-versions/builds/46811884#L3834

```
Error: Directory not empty - (/home/travis/build/saketkc/linuxbrew/Cellar/xz/5.2.0.reinstall, /home/travis/build/saketkc/linuxbrew/Cellar/xz/5.2.0)

```

Any clues?

**Comment by dpo** [comment]
I've had issues with `xz` before. Perhaps there's still another version of it lingering?

**Pull request closed without merging**